atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -205,15 +209,37 @@ public static SerializableTransactionManager createInMemory(Set<Schema> schemas)
     @JsonIgnore
     @Value.Derived
     public SerializableTransactionManager serializable() {
+        List<AutoCloseable> closeables = Lists.newArrayList();
```

Comment:
Could also do try-with-resources with a list that implements autocloseable and cut down on the code here, but probably isn't worth it.

Reply:
Nevermind, since we only want to clean them up in case of an exception.